eng/pipelines/test-windows-job-single-machine.yml

```diff
@@ -28,6 +28,15 @@ jobs:
   steps:
     - checkout: none
 
+    # A .NET 9 runtime is needed since our unit tests target net9.0
```

Comment:
I was wondering, where was the runtime used for testing coming from before this change?

Reply:
Can only assume the windows image has .Net 8.0. I know we used to install .Net 7.0 runtime in this manner until earlier this year.